app/src/main/java/com/framgia/soundclound/data/source/repository/GenresRepository.java

```diff
@@ -30,16 +30,16 @@
             GENRES_POP_BLAS, GENRES_DEEP_HOUSE, GENRES_COUNTRY,
             GENRES_ROCK, GENRES_EDM, GENRES_ELECTRONIC
     })
-public @interface ServiceName {
-    String GENRES_ALL = "all-music";
-    String GENRES_POP = "pop";
-    String GENRES_COUNTRY = "country";
-    String GENRES_POP_BLAS = "jazzblues";
-    String GENRES_DEEP_HOUSE = "deephouse";
-    String GENRES_ROCK = "rock";
-    String GENRES_EDM = "danceedm";
-    String GENRES_CLASSIC = "classical";
-    String GENRES_ELECTRONIC = "electronic";
+    public @interface ServiceName {
+        String GENRES_ALL = "all-music";
+        String GENRES_POP = "pop";
```

Comment:
ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.